docs/pywbemcli/generaloptions.rst

```diff
@@ -635,6 +635,29 @@ normally return nothing upon successful execution(ex. instance delete,
 instance enumerate that returns no CIM objects) to indicate the successful
 command completion.
 
+.. index:: triple: --connections-file; general options; connection-file
+
+.. _`--connections-file general option`:
+
+``--connections-file`` general option
+""""""""""""""""""""""""""""""""""""""
+
+The ``--connections-file``/``-C`` general option allows the user to select
+a file path for the connections file. It is the file path of a YAML file
+that defines server configurations by name.  The default if this option is
+not specified is to use the configuration file
+``pywbemcli_connection_definitions.yaml`` defined in the user home directory.
```

Comment:
1. c/configuration file/connections file/
2. The file is not "defined" in the user home directory, it simply is there. I.e. remove "defined".

Reply:
DONE